lib/Model/Attendee.php

```diff
@@ -59,7 +59,8 @@
  * @method int getLastAttendeeActivity()
  */
 class Attendee extends Entity {
-	public const ACTOR_USERS = 'users';
+	public const ACTOR_USERS = 'users';lib/Chat/SystemMessage/Listener.php
```

Comment:
Should be like this?

Reply:
No, it's copy paste fail, but psalm is complaining in this PR anyway and fixing it is only possible with a lot of duplicated docs